### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by NishiPhalke**

### CODE REVIEW

File: screen2.0/src/app/search/page.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ export default function Search({ searchParams }: { searchParams: { [key: string]
   const [opencCREs, setOpencCREs] = useState<{
     ID: string,
     region: { start: number, end: number, chrom: string },
-    linkedGenes: LinkedGenesData
+    linkedGenes?: LinkedGenesData
```
Comment (NishiPhalke) [inline screen2.0/src/app/search/page.tsx:130]: Made this optional for now. Need to change this

File: screen2.0/src/app/search/_ccredetails/ccredetails.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export const CcreDetails: React.FC<CcreDetailsProps> = ({ accession, region, bio
         />
       )}
       {page === 3 && <Ortholog accession={accession} assembly={assembly} />}      
-      {page === 4 && <GeneExpression assembly={assembly} genes={geneList} />}
+      {page === 4 && geneList && <GeneExpression assembly={assembly} genes={geneList} />}
```
Comment (NishiPhalke) [inline screen2.0/src/app/search/_ccredetails/ccredetails.tsx:51]: Not loading gene expression if Linkedgenes data is not available as a workaround

Comment (jpfisher72) [reply screen2.0/src/app/search/_ccredetails/ccredetails.tsx:51]: Thanks, this is a good workaround for now

File: screen2.0/src/app/search/page.tsx (hunk 1 above)
Comment (jpfisher72) [reply screen2.0/src/app/search/page.tsx:130]: Gotcha, I will change this

**Review by jpfisher72 — APPROVED** [review]

**Pull request merged**